src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py

```diff
@@ -251,11 +249,16 @@ def reportRequestStatus(self, request, status, message = None):
         """
         if message:
             self.sendMessage(request, str(message), 'info')
```

Comment:
What is this message? When is it used? Is it the messages we see in ReqMgr when there is some failure? Depending on the answer, we could use logdb for this one as well :)